### DISCUSSION TITLE
Custom Thumb Height in QScrollArea?

### DISCUSSION THREAD

**Discussion by zutamacka** [quasarframework/quasar#13526] [General - Components / Directives / etc]
Hello all; 

I would like to customize the thumb height in QScrollArea as it's too large for my needs - I'd like to cut it to about half the size (see https://i.imgur.com/mPQOVhQ.png). 

This is my css: 
`      thumbStyle: {
        right: '4px',
        borderRadius: '5px',
        backgroundColor: '#1BA9FF',
        width: '6px',
        opacity: 0.75,
      },
      barStyle: {
        right: '2px',
        borderRadius: '9px',
        backgroundColor: '#027be3',
        width: '9px',
        opacity: 0.2,
      },
`

I tried & failed:
- setting the height
- setting a transparent top&bottom border
- playing with paddings and margins

Please help..?

**Comment by pdanpdan** [comment]
make a codepen

**Reply by zutamacka** [reply]
Hello; sorry for the wait: https://codepen.io/zutamacka/pen/eYVeNvB

**Reply by pdanpdan** [reply]
It's not very clear for me what you want to achieve (because the height of the thumb should be related to the percent of height visible) but maybe this helps you: https://codepen.io/pdanpdan/pen/YzeEyPb?editors=0010

**Reply by zutamacka** [reply]
Thank you for the effort! :)

I'm trying to achieve what you basically did, only to have the thumb be able to travel all the way up and down the bar. This way, it's kind of unclear whether the user has reached the end of the listed options. 

> It's not very clear for me what you want to achieve (because the height of the thumb should be related to the percent of height visible) 


I understand this as a general idea. I'm confused with why the thumb remains so "long" in my original code, it's nearly 100% of the height visible. It seems to remain constant size regardless of the height visible: https://codepen.io/zutamacka/pen/MWQOyYw

**Reply by pdanpdan** [reply]
Ah, I understand what you mean - there is a minimum height/width for the thumb(vertical/horizontal) and this is 50px
So even if the thumb size calculated based on the ratio would be smaller than 50px you will still get a size of 50px

**Reply by zutamacka** [reply]
Thank you for your help; didn't know that...
